Coral/Coral.js

```diff
@@ -25,9 +35,13 @@ const CONNECTION_ORDER = [
   [0b0001, 0b0101, 0b0100, 0b0000],
 ];
 
-const TILES = SMALL_QUADS.map(
-  ({ i, j }, quad) => new CoralTile(quad, CONNECTION_ORDER[i][j])
-);
+const TILES = SMALL_QUADS.map(({ i, j }, quad) => {
```

Comment:
Add a button that when clicked exports the pattern as JSON. Maybe to a text area?

Reply:
Added a button that downloads the JSON as a file